dashboard/app/models/script.rb

```diff
@@ -1812,6 +1812,10 @@ def course_title
     unit_group.try(:localized_title)
   end
 
+  def unversioned?
+    version_year.blank? || version_year == 'unversioned'
```

Comment:
I poked around a bit more, and it looks like we actually have a constant for this!
```suggestion
    version_year.blank? || version_year == CourseVersion::UNVERSIONED
```

Reply:
Ah thanks for finding this!